test/extensions/filters/network/zookeeper_proxy/filter_test.cc

```diff
@@ -30,7 +31,21 @@ class ZooKeeperFilterTest : public testing::Test {
   ZooKeeperFilterTest() { ENVOY_LOG_MISC(info, "test"); }
 
   void initialize() {
-    config_ = std::make_shared<ZooKeeperFilterConfig>(stat_prefix_, 1048576, scope_);
+    std::chrono::milliseconds default_latency_threshold(100);
+    LatencyThresholdOverrideList latency_threshold_overrides;
+
+    initializeHelper(default_latency_threshold, latency_threshold_overrides);
+  }
+
+  void initialize(std::chrono::milliseconds default_latency_threshold,
+                  LatencyThresholdOverrideList& latency_threshold_overrides) {
+    initializeHelper(default_latency_threshold, latency_threshold_overrides);
+  }
+
+  void initializeHelper(std::chrono::milliseconds default_latency_threshold,
+                        LatencyThresholdOverrideList& latency_threshold_overrides) {
```

Comment:
Since there are lots of `initialize()` func call already in the tests without parameters, I created the overload funcs. Otherwise, I need to change each `initialize()` func call.

Reply:
Sorry, I don't follow.
Can something similar to the following be used:
```
void initialize(std::chrono::milliseconds default_latency_threshold = 100,
                       const LatencyThresholdOverrideList& latency_threshold_overrides = {}) {
    ....
}
```
(maybe needs a bit of massaging of the values after the `=`)

Reply:
Sorry, you are right, was thinking about Golang 🤦